### DISCUSSION TITLE
Audiogridder help - no gui on server

### DISCUSSION THREAD

**Discussion by riffwraith** [apohl79/audiogridder#1273] [General]
Hi

Just installed AG. The plug in opens fine on my host, but the server - on my W11 computer, does not show the GUI. It shows as running in the TM, but I see nothing. I tried changing the mon. resolution, but nothing. Should I be seeing something? If not, how to load a plug? Anyone have any ideas?

Thanks in advance.

**Comment by riffwraith** [comment]
Ok, it's working here - but with no gui. Until I connect using my host - then there's a gui. Strange, but I can't complain.

**Comment by apohl79** [comment]
you mean there is no tray icon for you?
